src/solvers/dg/2d/dg.jl

```diff
@@ -2548,6 +2548,14 @@ end
 @inline calc_dt(dg, cfl) = calc_dt(dg, cfl, uses_mpi(dg))
 function calc_dt(dg::Dg2D, cfl, uses_mpi::Val{false})
   min_dt = Inf
+
+  # FIXME: This should be implemented properly using another callback
+  #        or something else, cf.
+  #        https://github.com/trixi-framework/Trixi.jl/issues/257
+  if dg.equations isa AbstractIdealGlmMhdEquations
+    dg.equations.c_h = zero(dg.equations.c_h)
+  end
+
```

Comment:
```suggestion
```

I don't think we should change Taam anymore. Taal is fully verified and feature-complete, thus there is no need to keep updating Taam anymore. To me, Taam is dead, long live Taal.